src/js/components/returns/inbound/AddItemsPage.jsx

```diff
@@ -190,13 +192,8 @@ class AddItemsPage extends Component {
     return this.state.sortOrder;
   }
 
-  updateRow(values, index) {
-    const item = values.returnItems[index];
-    this.setState({
-      formValues: update(values, {
-        returnItems: { [index]: { $set: item } },
-      }),
-    });
+  updateFormValues(values) {
+    this.setState({ formValues: values });
```

Comment:
I have refactored this whole `updateRow` function since to me the logic that is inside seems redundant.
Maybe I am not seeing something so please point it out to me but it looks like it extracts `item` from `values` and later modifies `values` bu inserting `item` to the same index that it was previously extracted from.